tests/test_testPackage.py

```diff
@@ -60,3 +63,35 @@ def test_packageExtractWithUnicodePath(self):
       self.assertTrue(os.path.isdir(f"{tmp}/Assets"))
       self.assertTrue(os.path.isfile(f"{tmp}/Assets/テスト.txt"))
       self.assertEqual(open(f"{tmp}/Assets/テスト.txt").read(), "テスト, but with katakana!")
+
+  @patch('sys.stdout', new_callable=io.StringIO)
+  def test_packageExtractEscape(self, stdout):
+    '''should skip relative paths that are outside'''
+    #arrange
+    with tempfile.TemporaryDirectory() as tmp:
+      # testEscape.unitypackage - Should skip relative paths
+
+      #act
+      print(f"Extracting to {tmp}...")
+      extractPackage("./tests/testEscape.unitypackage", outputPath=tmp)
+
+      #assert
+      self.assertTrue(os.path.isdir(tmp))
+      self.assertTrue(not os.path.isfile(f"{tmp}/../escape.txt"))
+      self.assertTrue(re.search(r"outside", stdout.getvalue(), flags=re.IGNORECASE))
+
+  @patch('sys.stdout', new_callable=io.StringIO)
+  def test_packageExtractEscape2(self, stdout):
+    '''should skip absolute paths'''
+    #arrange
+    with tempfile.TemporaryDirectory() as tmp:
+      # testEscape2.unitypackage - Should skip absolute paths
+
+      #act
+      print(f"Extracting to {tmp}...")
+      extractPackage("./tests/testEscape2.unitypackage", outputPath=tmp)
+
+      #assert
+      self.assertTrue(os.path.isdir(tmp))
+      self.assertTrue(not os.path.isfile(f"/var/log/escape.txt"))
```

Comment:
I don't think `/var/log` is user-writeable on most systems. It might make more sense to use `/tmp` for the test file. The test would still fail if extractPackage() wasn't fixed, but from a `PermissionsError` thrown by `shutil.move` unless you have write access to `/var/log`.  

I wasn't sure if this test would also work for Windows, but after some testing it actually seems fine. Python seems to interpret `"/var/log/escape.txt"` as `C:\var\log\escape.txt` when given to function that take paths. Same goes for directories, so this should work fine.

Reply:
This seems to be correct, I'm not super familiar with *nix permissions but `/var/log` is indeed not writable by anything but owner. I'll switch to `/tmp`

Not too worried about running the tests on Windows, but maybe I should? Tests are more setup for my sanity/PRs/getting the status badge working

Reply:
This one is fixed